doc/dev/developer_guide/JaegerTracer.rst

```diff
@@ -0,0 +1,135 @@
+
```

Comment:
`s/JaegerTracer.rst/jaegertracing.rst`

- we would want to keep it neutral as a `intro and how to guide to using distributed tracing in Ceph` you may want to rephrase and add a `rgw tracing` section.
- can you turn steps to command users can directly copy and use. 
- can you add the example script directly, instead of link

Reply:
i think there should be 2 different docs:
* developer guide: give some background on architecture and explain how to trace new functions and paths
* user guide (new doc): info on how to turn tracing on/off (via config) and how to configure the tracing ecosystem

Reply:
noted